dependency/src/main/java/de/dagere/peass/execution/gradle/AnboxTestExecutor.java

```diff
@@ -81,8 +82,20 @@ private void updateAndroidManifest() {
 
    private void compileSources() {
       String wrapper = new File(folders.getProjectFolder(), EnvironmentVariables.fetchGradleCall()).getAbsolutePath();
+      List<String> gradleTasks = testTransformer.getConfig().getExecutionConfig().getAndroidGradleTasks();
 
-      ProcessBuilder builder = new ProcessBuilder(wrapper, "installDebug", "installDebugAndroidTest");
+      String[] processArgs = new String[1 + (gradleTasks != null ? gradleTasks.size() : 0)];
+      processArgs[0] = wrapper;
+
+      if (gradleTasks != null) {
+         int index = 1;
+
+         for (String task : gradleTasks) {
```

Comment:
This can be done using `System.arraycopy`, right? If yes, could you change it, if not, could you extract a method for adding the tasks to the `processArgs`?

Reply:
`System.arraycopy` is for arrays, here `gradleTasks` is a `List`, hence I extracted a new method as you suggested.